torch/_utils.py

```diff
@@ -317,6 +318,62 @@ def _rebuild_parameter(data, requires_grad, backward_hooks):
     return param
 
 
+def _rebuild_parameter_v2(data, requires_grad, backward_hooks, state):
```

Comment:
We should add a comment to `_rebuild_parameter` to say that it is now unused and only for BC.